### PR TITLE
PT-2193 Release Notes 3.5.2

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   Changelog
modified:   Makefile.PL
modified:   config/sphinx-build/conf.py
modified:   docs/release_notes.rst
new file:   docs/rn.3-5-2.txt

- [ ] The contributed code is licensed under GPL v2.0
- [ ] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [ ] Documention updated
- [ ] Test suite update

### CODE REVIEW

**Review by percona-csalguero — APPROVED** [review]

**Review by patrickbirch — APPROVED** [review]

File: Changelog [hunk 1]
```diff
@@ -1,5 +1,27 @@
 Changelog for Percona Toolkit
 
+v3.5.2 release 2023-03-28
+
```
Comment (svetasmirnova) [inline Changelog:4]:
```suggestion

+ * Improvement PT-188: Use percona.checksums in examples
+ * 
```

File: Changelog [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+* Improvement PT-2150: Fix tests for pt-mysql-summary
+* Improvement PT-2152: Fix tests for pt-show-grants
+* Improvement PT-2155: Fix tests for pt-upgrade
+* Improvement PT-2156: Fix tests for lib
+* Improvement PT-2159: Fix tests for pt-duplicate-key-checker
+* Improvement PT-2160: Fix tests for pt-online-schema-change
+* Improvement PT-2161: Fix tests for pt-slave-restart
+* Improvement PT-2164: pt-k8s-debug-collector does not have version flag
+* Improvement PT-2165: Actual Version, GoVersion, and Build for Go tools
+* Fixed bug   PT-2187: CVEs reported in Percona Toolkit version 5.3.1 related to GO
```
Comment (svetasmirnova) [inline Changelog]:
```suggestion
* Fixed bug   PT-215: Documentation fix
* Fixed bug   PT-1059: LP #1093972: Tools can't parse index names containing newlines
* Fixed bug   PT-1389: LP #1637859: [pt-variable-advisor] Docs Typos
* Fixed bug   PT-1594: Need correction in the Documentation - pt-mysql-summary
* Fixed bug   PT-1595: percona toolkit docs still reference Maatkit
* Fixed bug   PT-1642: Update PT docs to reference bug reporting location as Percona JIRA
* Fixed bug   PT-1683: List of supported distributions in the docs is outdated
* Fixed bug   PT-1700: pt-table-checksum REPLICATION_STOPPED exit status not documented
* Fixed bug   PT-1927: pt-k8s-debug-collector requires mysql client
* Fixed bug   PT-1930: pt-k8s-debug-collector should not attempt collecting PXC information when running against PSMDB
* Fixed bug   PT-2187: CVEs reported in Percona Toolkit version 5.3.1 related to GO
```

File: Changelog [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+* Fixed bug   PT-2084: Changes from PT-80 broke previous pt-stalk functionality
```
Comment (svetasmirnova) [inline Changelog]:
```suggestion
* Fixed bug   PT-2084: Changes from PT-80 broke previous pt-stalk functionality (Thanks Agustín Gallego)
```

File: Changelog [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+* Fixed bug   PT-2102: pt-mysql-summary should support !include* in config files
```
Comment (svetasmirnova) [inline Changelog]:
```suggestion
* Fixed bug   PT-2102: pt-mysql-summary should support !include* in config files (Thanks Yoann La Cancellera)
```

File: Changelog [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+* Fixed bug   PT-2110: Incorrect PTDEBUG output with --ignore-engines option (Thanks Kazuya Yokogawa)
+* Fixed bug   PT-2114: Incorrect casting of BIT columns by pt-archiver
+* Fixed bug   PT-2116: pt-osc --null-to-not-null description is wrong
```
Comment (svetasmirnova) [inline Changelog]:
```suggestion
* Fixed bug   PT-2116: pt-osc --null-to-not-null description is wrong (Thanks Yoann La Cancellera)
```

File: docs/release_notes.rst [hunk 6]
```diff
@@ -1,6 +1,36 @@
...
+* :jirabug:`PT-2160`: Fix tests for pt-online-schema-change
+* :jirabug:`PT-2161`: Fix tests for pt-slave-restart
+* :jirabug:`PT-2164`: pt-k8s-debug-collector does not have version flag
+* :jirabug:`PT-2165`: Actual Version, GoVersion, and Build for Go tools
+
+Bugs Fixed
+------------
+
+* :jirabug:`PT-2187`: CVEs reported in Percona Toolkit version 5.3.1 related to GO
+* :jirabug:`PT-2084`: Changes from PT-80 broke previous pt-stalk functionality
```
Comment (svetasmirnova) [inline docs/release_notes.rst]:
```suggestion
* :jirabug:`PT-2084`: Changes from PT-80 broke previous pt-stalk functionality (Thanks Agustín Gallego)
```

File: docs/rn.3-5-2.txt [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+* :jirabug:`PT-2159`: Fix tests for pt-duplicate-key-checker
+* :jirabug:`PT-2160`: Fix tests for pt-online-schema-change
+* :jirabug:`PT-2161`: Fix tests for pt-slave-restart
+* :jirabug:`PT-2164`: pt-k8s-debug-collector does not have version flag
+* :jirabug:`PT-2165`: Actual Version, GoVersion, and Build for Go tools
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PT-2084`: Changes from PT-80 broke previous pt-stalk functionality.
```
Comment (svetasmirnova) [inline docs/rn.3-5-2.txt]:
```suggestion
* :jirabug:`PT-2084`: Changes from PT-80 broke previous pt-stalk functionality. (Thanks Agustín Gallego)
```

File: docs/release_notes.rst [hunk 8]
```diff
@@ -1,6 +1,36 @@
...
+* :jirabug:`PT-2102`: pt-mysql-summary should support !include* in config files
```
Comment (svetasmirnova) [inline docs/release_notes.rst]:
```suggestion
* :jirabug:`PT-2102`: pt-mysql-summary should support !include* in config files (Thanks Yoann La Cancellera)
```

File: docs/release_notes.rst [hunk 9]
```diff
@@ -1,6 +1,36 @@
...
+* :jirabug:`PT-2110`: Incorrect PTDEBUG output with --ignore-engines option (Thanks to Kazuya Yokogawa for reporting this issue)
+* :jirabug:`PT-2114`: Incorrect casting of BIT columns by pt-archiver
+* :jirabug:`PT-2116`: pt-osc --null-to-not-null description is wrong
```
Comment (svetasmirnova) [inline docs/release_notes.rst]:
```suggestion
* :jirabug:`PT-2116`: pt-osc --null-to-not-null description is wrong (Thanks Yoann La Cancellera)
```

File: docs/rn.3-5-2.txt [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+* :jirabug:`PT-2102`: pt-mysql-summary should support !include* in config files.
```
Comment (svetasmirnova) [inline docs/rn.3-5-2.txt]:
```suggestion
* :jirabug:`PT-2102`: pt-mysql-summary should support !include* in config files. (Thanks Yoann La Cancellera)
```

File: docs/rn.3-5-2.txt [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+* :jirabug:`PT-2110`: Incorrect PTDEBUG output with --ignore-engines option.
+* :jirabug:`PT-2114`: Incorrect casting of BIT columns by pt-archiver.
+* :jirabug:`PT-2116`: pt-osc --null-to-not-null description is wrong.
```
Comment (svetasmirnova) [inline docs/rn.3-5-2.txt]:
```suggestion
* :jirabug:`PT-2116`: pt-osc --null-to-not-null description is wrong. (Thanks Yoann La Cancellera)
```

**Review by svetasmirnova — CHANGES_REQUESTED** [review]
Please review my suggestions for the ChangeLog. Feel free to add missed documentation bugs into Release Notes too.

File: Changelog (hunk 1 above)
Comment (alina-derkach-oaza) [reply Changelog:4]: Added

File: Changelog (hunk 2 above)
Comment (alina-derkach-oaza) [reply Changelog]: Updated

File: Changelog (hunk 4 above)
Comment (alina-derkach-oaza) [reply Changelog]: Updated

File: Changelog (hunk 5 above)
Comment (alina-derkach-oaza) [reply Changelog]: Updated

File: docs/release_notes.rst (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/release_notes.rst]: Updated

File: docs/rn.3-5-2.txt (hunk 7 above)
Comment (alina-derkach-oaza) [reply docs/rn.3-5-2.txt]: Updated

File: docs/release_notes.rst (hunk 8 above)
Comment (alina-derkach-oaza) [reply docs/release_notes.rst]: Updated

File: docs/release_notes.rst (hunk 9 above)
Comment (alina-derkach-oaza) [reply docs/release_notes.rst]: Updated

File: docs/rn.3-5-2.txt (hunk 10 above)
Comment (alina-derkach-oaza) [reply docs/rn.3-5-2.txt]: Updated

File: docs/rn.3-5-2.txt (hunk 11 above)
Comment (alina-derkach-oaza) [reply docs/rn.3-5-2.txt]: Updated

File: Changelog (hunk 3 above)
Comment (alina-derkach-oaza) [reply Changelog]: Updated

**Review by svetasmirnova — APPROVED** [review]

**Pull request merged**